atomic_reactor/odcs_util.py

```diff
@@ -69,6 +70,9 @@ def start_compose(self, source_type, source, packages=None, sigkeys=None, arches
         if packages:
             body['source']['packages'] = packages
 
+        if flags is not None:
+            body['source']['flags'] = flags
```

Comment:
Looking at odcs source code, `'flags`' behave like `'arches'` and should be directly in `'body'`, not nested under `'source'`: https://pagure.io/odcs/blob/master/f/client/odcs/client/odcs.py#_246-257

Reply:
The ODCS docs do not make that very clear:
https://pagure.io/odcs
will fix.

Reply:
Fixed.